### DISCUSSION TITLE
Needless background on transparent avatars for organizations

### DISCUSSION THREAD

**Discussion by ruivieira** [community/community#61338] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When using an image with a transparent background as an organization's avatar, it displays as intended in light themes.
However, when using a dark theme, the `background-color: rgba(255, 255, 255, 0.1)` in `.avatar` is much more noticeable, it is detrimental and it interferes with the image's intended design.

In my opinion, if users want to add a subtle background to the image, they should do so themselves.

**Comment by RobGeada** [comment]
This is how transparency works currently on the dark background:
<img width="549" alt="Screenshot 2023-07-20 at 10 40 53 AM" src="https://github.com/community/community/assets/20345051/8ffe08c0-e45b-4756-9c8f-45b4374ec691">

This is what it would look like with @ruivieira's suggestion:
![image](https://github.com/community/community/assets/20345051/b02afa87-9879-46d9-a8d1-824c2d83b0af)

**Comment by alexcreasy** [comment]
Agreed, you're implicitly messing with people's chosen design here. 

The "default" look you've chosen is easily achievable in 2 minutes with Inkscape -if- that was what the designer intended.

**Comment by setvik** [comment]
Agreed. Would be awesome to get this fixed.
